src/test/java/com/courtcanva/ccfranchise/services/FranchiseeServiceTest.java

```diff
@@ -84,7 +85,7 @@ public void setFranchiseeServiceUp() {
     }
 
     @Test
-    void shouldCreateStaffAndFranchiseeGetDto() {
+    void givenFranchiseeAndStaffPostDto_whenCreateFranchiseeAndStaff_shouldReturnStaffAndFranchiseeGetDto() throws MailingClientException {
```

Comment:
Check if `MailingClientException ` is neccesary. how to process this exception

Reply:
Done done